src/quo/components/button/view.cljs

```diff
@@ -56,7 +56,7 @@
 
 (defn button [{:keys [on-press disabled type theme before after
                       haptic-feedback haptic-type on-long-press on-press-start
-                      accessibility-label loading border-radius style]
+                      accessibility-label loading border-radius style test-ID]
```

Comment:
yeah, this one I'm not entirely sure about because I tried to used :test-id but it doesn't work.. perhaps I need to do "test-i-d or something. In any case the discrepancy between the two is somewhat advantageous as we clearly see it is not a standard prop.
it is a bit sore on the eyes though 😆 so I would be up for changing it in follow up work

Reply:
Yep, this is not necessarily difficult, but it's a reasonable amount of work that shouldn't be done in this PR. Ideally IMO, the plumbing layer (the functions that abstract React Native classes, like `rn/view`), should all translate Clojure idioms to RN, i.e. `:test-id` is perfectly valid and expected in Clojure, but it would be translated to `:testID` over there.

Leaking programming language idioms is usually something I'm worried about.